src/containers/articles/Article.js

```diff
@@ -5,8 +5,14 @@ import { getArticle, deleteArticle, toggleLike } from '../../actions/articleActi
 import Article from '../../components/articles/Article';
 
 class ArticlePage extends Component {
+  componentWillMount() {
+    const { articleData } = this.props; // eslint-disable-line
+    articleData.id = 0;
+  }
+
   componentDidMount() {
     const { articleData, getArticle, history } = this.props; // eslint-disable-line
+    articleData.id = 0;
     if (!articleData.id) {
```

Comment:
At what point is this condition `!articleData.id` going to be false?

Reply:
As soon as the page renders, the id false is what triggers the loader.